internal/runners/branch/list.go

```diff
@@ -33,35 +30,24 @@ func NewList(prime primeable) *List {
 func (l *List) Run() error {
 	logging.Debug("ExecuteList")
 
+	l.out.Print(
+		locale.Tl(
+			"branch_info",
+			"\nBranches allow you to create runtimes with different packages sets depending on your use case. Here are the branches in your current project.\n",
+		),
+	)
+
```

Comment:
Was this in the mock? Let's drop this. NONE of our other commands do this.

Reply:
Yes, again from the mock